marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-step-detail/mapping-step-detail.tsx

```diff
@@ -26,6 +26,7 @@ import arrayIcon from "../../../../assets/icon_array.png";
 import relatedEntityIcon from "../../../../assets/icon_related_entities.png";
 import CustomPageHeader from "../../page-header/page-header";
 import {clearSessionStorageOnRefresh, getViewSettings, setViewSettings} from "../../../../util/user-context";
+import {paginationOptions, tableColors} from "../../../../config/mapping.config";
```

Comment:
To display the colored tables correctly from the config file, you need to clone it, e.g.:

_in mapping.config.js_

export const mappingColors = ["#e4f1f4", "#f0f9f5",...

_in mapping-step-detail.tsx_

import {paginationOptions, mappingColors} from "../../../../config/mapping.config";
...
let tableColors = [...mappingColors];

Reply:
Fixed it.